### PR TITLE
Yaml logo size increased in readme.md file

### PR DESCRIPTION
**Pull request by AniqJaved**
## Purpose

The purpose of this Pull Request is to fix #<issue-number>

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ yes] I have read and understood the development best practices guidelines
- [ yes] My code follows the style guidelines of this project
- [yes ] I have performed a self-review of my own code
- [ yes] I have commented my code, particularly in hard-to-understand areas
- [ yes] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Screenshot required @AniqJaved

**Comment by AniqJaved** [comment]
image

**Comment by AniqJaved** [comment]
Done👍

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 > Used Programming Languages (HTML, CSS, JavaScript, Bootstrap, YAML) 
 
 <a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a>    <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>    <a href="https://www.w3schools.com/js/"><img src="https://img.icons8.com/color/48/000000/javascript--v1.png"/><a>    <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
-<a href="https://www.tutorialspoint.com/yaml/index.htm"><img width="32px" height="32px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/64px-Official_YAML_Logo.svg.png" alt="Official YAML Logo.svg"/></a>
+<a href="https://www.tutorialspoint.com/yaml/index.htm"><img width="50px" height="50px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/64px-Official_YAML_Logo.svg.png" alt="Official YAML Logo.svg"/></a>
```
Comment (YoshithaRathnayake) [inline README.md:17]: Make the icon size equal to the other icons @AniqJaved

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -14,7 +14,8 @@
 > Used Programming Languages (HTML, CSS, JavaScript, Bootstrap, YAML) 
 
 <a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a>    <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>    <a href="https://www.w3schools.com/js/"><img src="https://img.icons8.com/color/48/000000/javascript--v1.png"/><a>    <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
-<a href="https://www.tutorialspoint.com/yaml/index.htm"><img width="32px" height="32px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/64px-Official_YAML_Logo.svg.png" alt="Official YAML Logo.svg"/></a>
+<a href="https://www.tutorialspoint.com/yaml/index.htm"><img width="42px" height="34px" style="position: relative;
```
Comment (YoshithaRathnayake) [inline README.md:17]: Don't add css no use @AniqJaved

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+<style type="text/css" rel="stylesheet">
```
Comment (YoshithaRathnayake) [inline README.md:1]: Do not add styles to the readme.md file its useless @AniqJaved

**Comment by AniqJaved** [comment]
Kindly check the new commit @YoshithaRathnayake

**Comment by AniqJaved** [comment]
image

File: README.md [hunk 4]
```diff
@@ -21,8 +15,23 @@
 
 
 
-<a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a>    <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>    <a href="https://www.w3schools.com/js/"><img src="https://img.icons8.com/color/48/000000/javascript--v1.png"/><a>    <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
-<a href="https://www.tutorialspoint.com/yaml/index.htm"><img class="yaml-logo" width="42px" height="34px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/64px-Official_YAML_Logo.svg.png" alt="Official YAML Logo.svg"/></a>
+<p align="left">
```
Comment (YoshithaRathnayake) [inline README.md:18]: Can you remove this because it is not needed cuz its already left aligned @AniqJaved

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by AniqJaved** [comment]
Done👍

File: README.md [hunk 5]
```diff
@@ -13,8 +13,25 @@
 
 > Used Programming Languages (HTML, CSS, JavaScript, Bootstrap, YAML) 
 
-<a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a>    <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>    <a href="https://www.w3schools.com/js/"><img src="https://img.icons8.com/color/48/000000/javascript--v1.png"/><a>    <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
-<a href="https://www.tutorialspoint.com/yaml/index.htm"><img width="32px" height="32px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/64px-Official_YAML_Logo.svg.png" alt="Official YAML Logo.svg"/></a>
+
+
+<p>
```
Comment (YoshithaRathnayake) [inline README.md:18]: Actually no need of `<p>` tag also remove them @AniqJaved

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (AniqJaved) [reply README.md:18]: It is the standard procedure of tag provision in html. Further updates will be based on the scope of this tag.

Comment (YoshithaRathnayake) [reply README.md:18]: Ok then

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @AniqJaved

**Pull request merged**